integrations/docker/images/chip-build/Dockerfile

```diff
@@ -184,14 +232,30 @@ RUN set -x \
 #
 # This is not a CHIP dependency directly, but used by CI
 ENV CHIP_NODE_VERSION=v16.13.2
-RUN set -x \
-    && mkdir node_js \
-    && cd node_js \
-    && wget https://nodejs.org/dist/$CHIP_NODE_VERSION/node-$CHIP_NODE_VERSION-linux-x64.tar.xz \
-    && tar xfvJ node-$CHIP_NODE_VERSION-linux-x64.tar.xz \
-    && mv node-$CHIP_NODE_VERSION-linux-x64 /opt/ \
-    && ln -s /opt/node-$CHIP_NODE_VERSION-linux-x64 /opt/node \
-    && ln -s /opt/node/bin/* /usr/bin \
-    && cd .. \
-    && rm -rf node_js \
-    && : # last line
+RUN case ${TARGETPLATFORM} in \
+        "linux/amd64") \
+            set -x \
```

Comment:
I wish there were a way to reduce the duplication here by storing the tarball name in a variable.  Is there?  Similar for the bits above.